package.json

```diff
@@ -36,20 +36,20 @@
     "react-scripts": "^4.0.3",
     "redux": "^4.0.1",
     "reselect": "^4.0.0",
-    "scroll-to-element": "^2.0.2",
-    "stacktrace-js": "^2.0.0",
+    "scroll-to-element": "^2.0.3",
+    "stacktrace-js": "^2.0.2",
     "styled-components": "^4.3.2",
     "styled-icons": "^8.1.0",
     "typesafe-actions": "^4.4.2",
     "typescript": "^4.2.0",
     "url": "^0.11.0",
-    "uuid": "^3.4.0",
-    "weak-map": "^1.0.5",
-    "workbox-core": "^5.1.3",
-    "workbox-expiration": "^5.1.3",
-    "workbox-precaching": "^5.1.3",
-    "workbox-routing": "^5.1.3",
-    "workbox-strategies": "^5.1.3"
+    "uuid": "^7.0.3",
```

Comment:
the change to the UUID package has been problematic for us because they changed their module format -- if it works its fine, but if you get strange issues check this

Reply:
v8 and v9 didn't work, but v7 was ok.